client/landing/gutenboarding/domain-suggestions/constants.ts

```diff
@@ -0,0 +1 @@
+export const STORE_KEY = 'automattic/domain-suggestions';
```

Comment:
Or 
```suggestion
export const STORE_KEY = 'automattic/domains/suggestions';
```

? In case we wanna namespace more closely to existing (hierarchical) Calypso Redux state.

Reply:
Okay, I'll apply the diff :+1: